tasks/client.yml

```diff
@@ -18,16 +23,23 @@
 - name: Copy TA key and CA certificate
   copy:
     src: "{{ openvpn_client_certs_dir }}/{{ openvpn_server }}/{{ item }}"
-    dest: "{{ openvpn_dir }}/"
+    dest: "{{ openvpn_dir }}/{{ openvpn_server }}/"
   with_items:
     - ca.crt
     - ta.key
   notify: restart openvpn clients
 
-- name: Enable openvpn client
+- name: Enable IP forwarding
+  sysctl:
+    name: net.ipv4.ip_forward
+    value: 1
+    sysctl_set: yes
```

Comment:
Unfortunately there isn't a convenient common place to apply this for client and server (but not other hosts).  Duplicating it in the task files for each is fine, we aren't making a work of art here.